testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -1,76 +1,85 @@
+#!/usr/bin/env python
+
 import numpy as np
 import xarray as xr
 import matplotlib
 matplotlib.use('Agg')
 import matplotlib.pyplot as plt
-import os
 
 dpi = 200
 
+
 def rmse(resTag):
-# resTag is the resolution to compute RMSE
-# sliceTag is the time slice, assumes 1 hour output and same for all cases
-    fid = open('../../../{}/default/init_step/namelist.ocean'.format(resTag),'r')
+    """
+    resTag is the resolution to compute RMSE
+    """
```

Comment:
Better to use a docstring than a simple comment.  It can be parsed into documentation later on (if formatted properly). and it is easier to recognize and read.